navigation_and_routing/lib/src/data/author.dart

```diff
@@ -7,7 +7,7 @@ import 'book.dart';
 class Author {
   final int id;
   final String name;
-  final List<Book> books = <Book>[];
+  final List<Book> books = [];
```

Comment:
The pattern we've used elsewhere is:

`final books = <Book>[];`

It's slightly more concise.

Reply:
Thanks. Yes, it is now better.